packages/react-scripts/package.json

```diff
@@ -21,40 +21,40 @@
     "react-scripts": "./bin/react-scripts.js"
   },
   "dependencies": {
-    "autoprefixer": "7.1.0",
-    "babel-core": "6.24.1",
+    "autoprefixer": "7.1.1",
+    "babel-core": "6.25.0",
     "babel-eslint": "7.2.3",
     "babel-jest": "20.0.3",
     "babel-loader": "7.0.0",
     "babel-preset-react-app": "^3.0.0",
     "babel-runtime": "6.23.0",
-    "case-sensitive-paths-webpack-plugin": "1.1.4",
+    "case-sensitive-paths-webpack-plugin": "2.1.1",
```

Comment:
IIRC we purposely remained on 1.x because when we upgraded to 2.x previously it created an issue for Windows users.

I'm not sure if it was resolved yet: https://github.com/Urthen/case-sensitive-paths-webpack-plugin/issues/16